apps/client/src/features/rundown/event-block/EventBlock.module.scss

```diff
@@ -54,7 +54,16 @@ $skip-opacity: 0.1;
   }
 
   &.past:not(.skip) {
```

Comment:
nice!

if what we are trying to do is stop the opacity in an element, could we make it explicit that way?
```
&.past:not(.skip) {
  opacity: 0.4;

  .elementWithoutOpacity {
    opacity: 1;
  }
}
```

Reply:
for some weird reason this doesn't work. The element i'm sparing with the opacity is the .eventActions

if i write it like you proposed, it doesn't overwrite the opacity on these elements. Even with !Important

someone who is smarter with css than me could probably figure this out.

Reply:
hmm, probably, but probably not worth losing too much sleep over it
Would you mind adding a comment as to why this is necessary like?
/* we stop the eventActions from having opacity to fix issue with dropdown drawing order */

Reply:
all done